catalog/spatial/csw/spatial-csw-schema-bindings/src/main/resources/gmd/record.xsd

```diff
@@ -0,0 +1,646 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this should be created dynamically not statically. A client can request 1 to N schemas from a DescribeRecord request.